specification/mongocluster/DocumentDB.MongoCluster.Management/tspconfig.yaml

```diff
@@ -20,6 +20,18 @@ options:
     namespace: "com.azure.resourcemanager.mongocluster"
     service-name: "Mongo Cluster"
     examples-directory: "examples"
+  "@azure-tools/typespec-ts":
```

Comment:
Note: no need to change `emit`, since the `azure-sdk-for-js`'s script will do it for us